TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -43,30 +45,24 @@ actions.cds+=/variable,name=deathmark_kingsbane_condition,value=!talent.kingsban
 actions.cds+=/variable,name=deathmark_condition,value=!stealthed.rogue&buff.slice_and_dice.remains>5&dot.rupture.ticking&buff.envenom.up&!debuff.deathmark.up&variable.deathmark_ma_condition&variable.deathmark_kingsbane_condition
 # Usages for various special-case Trinkets and other Cantrips if applicable
 actions.cds+=/call_action_list,name=items
-## Invoke Externals to Deathmark
-## actions.cds+=/invoke_external_buff,name=power_infusion,if=dot.deathmark.ticking
 # Cast Deathmark if the target will survive long enough
 actions.cds+=/deathmark,if=(variable.deathmark_condition&target.time_to_die>=10)|boss&fight_remains<=20
 # Check for Applicable Shiv usage
-actions.cds+=/call_action_list,name=shiv
+actions.cds+=/call_action_list,name=shiv,strict=1,if=cooldown.shiv.ready
```

Comment:
This can cause Shiv to suddenly pop up in the recommendations, since we're explicitly disallowing Shiv from checking future usage.  I.e., if Shiv's CD has 0.4s remaining and the GCD has 0.8s remaining, this strict/if combo will avoid going to going to the Shiv action list until 0.4s have actually passed, instead of forecasting an 0.4s wait.

Reply:
Makes sense, will remove strict and keep condition?